code/modules/mob/language/monkey.dm

```diff
@@ -1,58 +1,13 @@
-/datum/language/human/monkey
-	name = "Chimpanzee"
+/datum/language/human/animal
+	name = LANGUAGE_ANIMAL
 	desc = "A collection of ooking noises made by monkeys"
-	speech_verb = "chimpers"
-	ask_verb = "chimpers"
-	exclaim_verb = "screeches"
+	speech_verb = "says"
+	ask_verb = "inquires"
+	exclaim_verb = "yells"
 	key = "C"
-	syllables = list("ook","eek")
+	syllables = list("chrp","tweet","squick","croak","hiss","gronk","meow","mew","bark", "woof", "bowwow", "yap", "arf") //This is a generalized animal language. If spoken by an animal, it comes out with the mob noises. This only occurs when spoken by a carbon (human).
```

Comment:
Wouldn't that mean that monkey species also will have all the noises instead of ones only relevant to their monkey languages?

Reply:
Correct. It's an edge case, however. Monkeys don't speak on their own so it'd be a player controlled monkey that would make these sounds, and player-controlled monkeys aren't able to be obtained normally.